### PR TITLE
docs(getting-started): Overhaul

### PR DESCRIPTION
**Pull request by adnanrahic**
This PR overhauls the getting started section of the docs.

## Changes

-

## Fixes

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: docs/docs/getting-started/cli-installation-reference.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# CLI Installation Reference
+
+This page contains a reference of all options how to install Tracetest CLI.
```
Comment (jfermi) [inline docs/docs/getting-started/cli-installation-reference.md:3]:
```suggestion
This page contains a reference of all options for installing Tracetest CLI.
```

File: docs/docs/getting-started/detailed-installation.md [hunk 2]
```diff
@@ -1,76 +1,18 @@
 # Detailed Instructions on Installing Tracetest Using the CLI
 
-Tracetest has a command line interface (CLI) which includes an **install wizard** that helps to install the Tracetest server into Docker or Kubernetes. The CLI can also be used to run tests, download or upload tests, and manage much of the capability of Tracetest.
+This page contains a reference of all options how to install Tracetest CLI, and how to use it to install Tracetest Server.
```
Comment (jfermi) [inline docs/docs/getting-started/detailed-installation.md:3]:
```suggestion
This page contains a reference of all options for installing Tracetest CLI, and how to use it to install Tracetest Server.
```

File: docs/docs/getting-started/detailed-installation.md [hunk 3]
```diff
@@ -152,8 +94,8 @@ In this step, you can choose to install just Tracetest alone or install it with
 
 ```text title="Expected output:"
 Do you have OpenTelemetry based tracing already set up, or would you like us to install a demo tracing environment and app? [type to search]: 
-> I have a tracing environment already. Just install Tracetest
-  Just learning tracing! Install Tracetest, OpenTelemetry Collector and the sample app.
+  I have a tracing environment already. Just install Tracetest
```
Comment (jfermi) [inline docs/docs/getting-started/detailed-installation.md:97]:
```suggestion
  I have a tracing environment already - Just install Tracetest.
```

File: docs/docs/getting-started/detailed-installation.md [hunk 4]
```diff
@@ -152,8 +94,8 @@ In this step, you can choose to install just Tracetest alone or install it with
...
+> Just learning tracing! Install Tracetest, OpenTelemetry Collector and the sample app.
```
Comment (jfermi) [inline docs/docs/getting-started/detailed-installation.md:98]:
```suggestion
> Just learning tracing? - Install Tracetest, OpenTelemetry Collector and the sample app.
```

File: docs/docs/getting-started/installation.mdx [hunk 5]
```diff
@@ -11,15 +11,35 @@ import GtagInstallCliTabs from '@site/src/components/GtagInstallCliTabs';
...
+Read the CLI installation reference [here](./detailed-installation).
+:::
+
+## Install the Tracetest Server
+
+Tracetest runs as a standalone container. It runs a Server and exposes a Web UI on port `11633`.
+
+You have two options to install Tracetest Server:
+
+- Using the Tracetest CLI to guide your installation in Docker and Kubernetes
```
Comment (jfermi) [inline docs/docs/getting-started/installation.mdx:38]:
```suggestion
- Using the Tracetest CLI to guide your installation in Docker and Kubernetes.
```

File: docs/docs/getting-started/installation.mdx [hunk 6]
```diff
@@ -11,15 +11,35 @@ import GtagInstallCliTabs from '@site/src/components/GtagInstallCliTabs';
...
+- Using the official [Helm chart](https://github.com/kubeshop/helm-charts/tree/main/charts/tracetest)
```
Comment (jfermi) [inline docs/docs/getting-started/installation.mdx:39]:
```suggestion
- Using the official [Helm chart](https://github.com/kubeshop/helm-charts/tree/main/charts/tracetest).
```

File: docs/docs/getting-started/installation.mdx [hunk 7]
```diff
@@ -36,30 +56,72 @@ tracetest server install
...
-:::note
-Follow the prompts and continue with all the default settings.
-This will generate a `./tracetest/` directory that contains a `docker-compose.yaml` and two more config files. One for Tracetest and one for OpenTelemetry collector.
+Choose to install Tracetest with the OpenTelemetry Collector and the [Pokeshop](../live-examples/pokeshop/overview.md) sample app.
 
-To see the output of this command, view [this sample setup](https://github.com/kubeshop/tracetest/tree/main/examples/tracetest-jaeger) from our GitHub examples.
-:::
+```text title="Expected output:"
+Do you have OpenTelemetry based tracing already set up, or would you like us to install a demo tracing environment and app? [type to search]: 
+  I have a tracing environment already. Just install Tracetest
```
Comment (jfermi) [inline docs/docs/getting-started/installation.mdx:63]:
```suggestion
  I have a tracing environment already. - Just install Tracetest.
```

File: docs/docs/getting-started/installation.mdx [hunk 8]
```diff
@@ -36,30 +56,72 @@ tracetest server install
...
+> Just learning tracing! Install Tracetest, OpenTelemetry Collector and the sample app.
```
Comment (jfermi) [inline docs/docs/getting-started/installation.mdx:64]:
```suggestion
> Just learning tracing? - Install Tracetest, OpenTelemetry Collector and the sample app.
```

File: docs/docs/getting-started/installation.mdx [hunk 9]
```diff
@@ -71,40 +133,116 @@ Start Docker Compose from the directory where you ran `tracetest server install`
...
 
-<CodeBlock
-  language="bash"
-  title="Condensed expected output from the Tracetest CLI:"
->
-{`export POD_NAME=$(kubectl get pods --namespace demo -l "app.kubernetes.io/name=pokemon-api,app.kubernetes.io/instance=demo" -o jsonpath="{.items[0].metadata.name}")
+```text title="Expected output:"
+Do you have OpenTelemetry based tracing already set up, or would you like us to install a demo tracing environment and app? [type to search]: 
+  I have a tracing environment already. Just install Tracetest
+> Just learning tracing! Install Tracetest, OpenTelemetry Collector and the sample app.
```
Comment (jfermi) [inline docs/docs/getting-started/installation.mdx:141]:
```suggestion
> Just learning tracing? - Install Tracetest, OpenTelemetry Collector and the sample app.
```

File: docs/docs/getting-started/no-otel.mdx [hunk 10]
```diff
@@ -0,0 +1,184 @@
...
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+No worries! You can get started with no code changes at all!
+
+This page will explain getting started with OpenTelemetry:
+
+- Injecting auto instrumentation with **no code changes**
+- Auto instrumentation with limited code changes
```
Comment (jfermi) [inline docs/docs/getting-started/no-otel.mdx:16]:
```suggestion
- Auto instrumentation with limited code changes.
```

Comment (jfermi) [reply docs/docs/getting-started/no-otel.mdx:16]:
```suggestion
- Auto instrumentation with limited code changes.
```

File: docs/docs/getting-started/no-otel.mdx [hunk 11]
```diff
@@ -0,0 +1,184 @@
...
+- Manual instrumentation with code changes
```
Comment (jfermi) [inline docs/docs/getting-started/no-otel.mdx:17]:
```suggestion
- Manual instrumentation with code changes.
```

File: docs/docs/getting-started/open.md [hunk 12]
```diff
@@ -0,0 +1,140 @@
...
+
+The **Trace** tab shows the entire distributed trace for debugging and a trace analysis score.
+
+![distributed trace and trace analysis score](https://res.cloudinary.com/djwdcmwdz/image/upload/v1688476460/docs/screely-1688476455986_q24aa2.png)
+
+### Test
+
+The **Test** tab shows span attributes. Here you add test specs and assertions on attribute values. You also get a test snippets out-of-the-box for common test cases.
+
+In the sample below you see how to assert that all database spans return in less than `100ms`.
```
Comment (jfermi) [inline docs/docs/getting-started/open.md:76]:
```suggestion
In the sample below, you see how to assert that all database spans return in less than `100ms`.
```

File: docs/docs/index.md [hunk 13]
```diff
@@ -1,35 +1,42 @@
...
-- Use your preferred trace back-end, like Jaeger or Tempo or OpenTelemetry Collector.
+- Define tests and assertions against every single microservice that a trace goes through.
+- Work with your existing distributed tracing solution, allowing you to build tests based on your already instrumented system.
 - Define multiple transaction triggers, such as a GET against an API endpoint, a GRPC request, etc.
-- Return both the response data and a full trace.
-- Define assertions against both the response and trace data, ensuring both your response and the underlying processes worked correctly, quickly and without errors.
-- Save tests.
-- Run the tests manually or via CI build jobs with the Tracetest CLI.
+- Define assertions against both the response and trace data, ensuring both your response and the underlying processes worked correctly, quickly, and without errors.
+- Save and run the tests manually or via CI build jobs with
```
Comment (jfermi) [inline docs/docs/index.md:25]: 'CI build jobs with...' what goes here?

File: docs/docs/index.md [hunk 14]
```diff
@@ -1,35 +1,42 @@
...
 
 New to trace-based testing? Read more about the concepts [here](./concepts/what-is-trace-based-testing).
 
-## Prerequisites
+## Architecture
+
+Here you can see how Tracetest interacts with a system under test.
 
-You need to add [OpenTelemetry instrumentation](https://opentelemetry.io/docs/instrumentation/) to your code and configure sending traces to a trace data store, or Tracetest directly, to benefit from Tracetest's trace-based testing.
+1. Trigger a test and generate a trace response
```
Comment (jfermi) [inline docs/docs/index.md:33]:
```suggestion
1. Trigger a test and generate a trace response.
```

File: docs/docs/index.md [hunk 15]
```diff
@@ -1,35 +1,42 @@
...
+2. Fetch traces to render and analyze them
```
Comment (jfermi) [inline docs/docs/index.md:34]:
```suggestion
2. Fetch traces to render and analyze them.
```

File: docs/docs/index.md [hunk 16]
```diff
@@ -1,35 +1,42 @@
...
+3. Add assertions to traces
```
Comment (jfermi) [inline docs/docs/index.md:35]:
```suggestion
3. Add assertions to traces.
```

File: docs/docs/index.md [hunk 17]
```diff
@@ -1,35 +1,42 @@
...
+4. See test results
```
Comment (jfermi) [inline docs/docs/index.md:36]:
```suggestion
4. See test results.
```

File: docs/docs/index.md [hunk 18]
```diff
@@ -1,35 +1,42 @@
...
+5. Run tests as part of CI/CD pipelines
```
Comment (jfermi) [inline docs/docs/index.md:37]:
```suggestion
5. Run tests as part of CI/CD pipelines.
```

**Review by jfermi — COMMENTED** [review]
Some editing suggestions.

File: docs/docs/index.md (hunk 13 above)
Comment (adnanrahic) [reply docs/docs/index.md:25]: We can drop the 'with' entirely.

**Comment by adnanrahic** [comment]
Thanks! Updated per the suggestions!

**Pull request merged**